handlers_global.go

```diff
@@ -61,6 +62,29 @@ func handleImport(s *Server) http.Handler {
 			return
 		}
 
+		if len(jsonMetrics) == 0 {
+			const msg = "Received empty /import request"
+			http.Error(w, msg, http.StatusBadRequest)
+			innerLogger.WithError(err).Error(msg)
+			return
+		}
+
+		var nonEmpty bool
```

Comment:
Ah! I can see that this would be confusing. The reason for choosing `nonEmpty` was that I want the zero value for the variable (`false`) to be the correct default. I'm adding some comments around here to make it easier to parse; if that's still confusing, I can try and think of a different naming scheme that doesn't involve a negative.